README.md

```diff
@@ -1,72 +1,41 @@
 # Microsoft Azure Active Directory Authentication Library (ADAL) for Python
+=====================================
 
-The ADAL for python library makes it easy for python applications to authenticate to AAD in order to access AAD protected web resources.
+[![Build Status](https://travis-ci.org/AzureAD/azure-activedirectory-library-for-python.svg?branch=dev)](https://travis-ci.org/AzureAD/azure-activedirectory-library-for-python)
+[![Documentation Status](https://readthedocs.org/projects/adal-python/badge/?version=latest)](https://adal-python.readthedocs.io/en/latest/?badge=latest)
 
-## Usage
+|[Getting Started](https://github.com/AzureAD/azure-activedirectory-library-for-python/wiki)| [Docs](https://aka.ms/aaddev)| [Samples](https://github.com/azure-samples?query=active-directory)| [Support](README.md#community-help-and-support)
+| --- | --- | --- | --- |
 
-### Install
 
-To support 'service principal' with certificate, ADAL depends on the 'cryptography' package. For smooth installation, some suggestions:
+The ADAL for Python library enables python applications to authenticate with Azure AD and get tokens to access Azure AD protected web resources.
 
-* For Windows and macOS
+You can learn in detail about ADAL Python functionality and usage documented in the [Wiki](https://github.com/AzureAD/azure-activedirectory-library-for-python/wiki).
 
-  Upgrade to the latest pip (8.1.2 as of June 2016) and just do `pip install adal`.
+## Versions
+Current version - 0.6.0
 
-* For Linux
+Minimum recommended version - 0.6.0
 
-  Upgrade to the latest pip (8.1.2 as of June 2016).
+You can find the changes for each version under [Releases](https://github.com/AzureAD/azure-activedirectory-library-for-python/releases).
 
-  You'll need a C compiler, libffi + its development headers, and openssl + its development headers.
-  Refer to [cryptography installation](https://cryptography.io/en/latest/installation/)
+> Note: Changes on 'client_id' and 'resource' arguments after 0.1.0
```

Comment:
Historically, these part showed up early in this document; and then in [one commit](https://github.com/AzureAD/azure-activedirectory-library-for-python/commit/6aaa7c718fdf46c89cd7cc848c9915b1edefed67#diff-04c6e90faac2675aa89e2176d2eec7d8) we purposely move it to the bottom of this document, and the reasoning was documented in that commit.

Reply:
@rayluo Thanks for the context. I placed it under versions since it is information related to a specific version and it is a grey text note to draw little attention. But if this is mostly historical content, is it still relevant and should we remove it instead?

Reply:
@navyasric You raised a very good point! At a hindsight, those paragraph "Changes ... after 0.1.0" really should belong to its release note. I've just updated the [release note on version 0.3.0](https://github.com/AzureAD/azure-activedirectory-library-for-python/releases/tag/0.3.0), you can go ahead to delete this paragraph from README now. Thank you!